blueocean-dashboard/src/main/js/creation/CreatePipelineStepsRenderer.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Created by cmeyers on 10/17/16.
+ */
+import React, { PropTypes } from 'react';
+import Extensions from '@jenkins-cd/js-extensions';
+import VerticalStep from './VerticalStep';
+
+const Sandbox = Extensions.SandboxedComponent;
```

Comment:
From jsdoc:  "Component that renders any passed-in children via a separate ReactDOM.render call. Useful for rendering untrusted React elements: errors will be trapped and displayed so the main UI isn't broken."